.jenkins/pytorch/test.sh

```diff
@@ -518,8 +533,9 @@ fi
 if [[ "${BUILD_ENVIRONMENT}" == *backward* ]]; then
   test_backward_compatibility
   # Do NOT add tests after bc check tests, see its comment.
-elif [[ "${BUILD_ENVIRONMENT}" == *xla* || "${JOB_BASE_NAME}" == *xla* ]]; then
+elif [[ "${TEST_CONFIG}" == *xla* ]]; then
   install_torchvision
+  build_xla
   test_xla
```

Comment:
Since we are running test_xla right after build_xla . Why do we need to clone_pytorch_xla() again inside test_xla ? Also which method is now running the tests in test_xla(), sorry its not clear.

Reply:
It's actually a no-op since it checks for the directories existence

Reply:
`test_xla` is still a function within `.jenkins/pytorch/test.sh` it doesn't show up in this diff since it was mostly unmodified

Reply:
Got ti Thanks!